docs/user/dashboard/lens.asciidoc

```diff
@@ -370,3 +375,9 @@ Here's a short list of few different aspects to check:
 ** If a custom `Number` configuration is used, check that the color stop values are covering the current data range.
 
 ** If a `Percent` configuration is used, and the data range changes, the colors displayed are affected.
+
+[float]
+[[is-it-possible-to-show-icons-in-datatable]]
+===== Is it possible to show icons in datatable?
+
+It is possible to show icons <<managing-index-patterns, via field formatter>>.
```

Comment:
```suggestion
You can display icons with <<managing-index-patterns, field formatter>> in data tables.
```

Reply:
Can you add an example for when users would want to display an icon in a data table?